scripts/install_extended.sh

```diff
@@ -125,29 +130,61 @@ case "$(uname -s | tr '[:upper:]' '[:lower:]')" in
+tag=$(ghExtractTag hazelcast/hazelcast-commandline-client)
+releaseUrl=$(printf "https://github.com/hazelcast/hazelcast-commandline-client/releases/download/v%s/hz-cli_%s_%s" "$tag" "$tag" "$bin_id")
 
-curl -L "$finalUrl" > "hz-cli"
+curl -L --silent "$releaseUrl" --output "hz-cli"
+chmod +x "./hz-cli"
 
 case "$(printf "${SHELL##*bin\/}")" in
     "zsh")
+        if [ ! -d "$HOME/.local" ]; then
+            mkdir $HOME/.local
+        fi
+        if [ ! -d "$HOME/.local/bin" ]; then
+            mkdir $HOME/.local/bin
+        fi
+        if [[ ! "$(cat $HOME/.zshrc)" == *"$(echo "export PATH=$HOME/.local/bin:$PATH")"* ]]; then
+            echo "export PATH=$HOME/.local/bin:$PATH" >> $HOME/.zshrc
+        fi
+        if [ ! -f "$HOME/.local/bin/hz-cli" ]; then
+            mv ./hz-cli $HOME/.local/bin/hz-cli
+        fi
         completionUrl=$(printf "https://raw.githubusercontent.com/hazelcast/hazelcast-commandline-client/main/extras/%s" "zsh_completion.sh")
-        curl "$completionUrl" --output "./zsh_completion.sh"
-        echo "autoload -U compinit; compinit" >> ~/.zshrc;
-        cat "./zsh_completion.sh" > "${fpath[1]}/_hz-cli"
+        curl --silent "$completionUrl" --output $HOME/.zsh_completion.sh
+        if [[ ! "$(cat $HOME/.zshrc)" == *"$(echo "autoload -U compinit; compinit")"* ]]; then
+            echo "autoload -U compinit; compinit" >> $HOME/.zshrc
+        fi
+        cat $HOME/.zsh_completion.sh > "${fpath[1]}/_hz-cli"
+        rm -rf $HOME/.zsh_completion.sh
     ;;
     "bash")
+        if [ ! -d "$HOME/.local" ]; then
+            mkdir $HOME/.local
+        fi
+        if [ ! -d "$HOME/.local/bin" ]; then
+            mkdir $HOME/.local/bin
+        fi
+        if [[ ! "$(cat $HOME/.bashrc)" == *"$(echo "export PATH=$HOME/.local/bin:$PATH")"* ]]; then
+            echo "export PATH=$HOME/.local/bin:$PATH" >> $HOME/.bashrc
+        fi
+        if [ ! -f "$HOME/.local/bin/hz-cli" ]; then
+            mv ./hz-cli $HOME/.local/bin/hz-cli
+        fi
+        if [ ! -d "$HOME/.bash_completion.d" ]; then
+            mkdir $HOME/.bash_completion.d
+        fi
         completionUrl=$(printf "https://raw.githubusercontent.com/hazelcast/hazelcast-commandline-client/main/extras/%s" "bash_completion.sh")
-        curl "$completionUrl" --output "./bash_completion.sh"
-        if [ "$(uname)" == "Darwin" ]; then
-            cat "./bash_completion.sh" > /usr/local/etc/bash_completion.d/hz-cli
-        elif [ "$(expr substr $(uname -s) 1 5)" == "Linux" ]; then
-            cat "./bash_completion.sh" > /etc/bash_completion.d/hz-cli
+        curl --silent "$completionUrl" --output $HOME/.bash_completion.d/hz-cli
+        if [[ ! "$(cat $HOME/.bashrc)" == *"$(echo "$bashrcAddition")"* ]]; then
+            echo "$bashrcAddition" >> $HOME/.bashrc
         fi
+        source $HOME/.bashrc
```

Comment:
This may have unintended side-effects. Let's revert to displaying a message to the user which tells starting the a new session or terminal.

Reply:
Done!